### DISCUSSION TITLE
最高音质优先设置建议

### DISCUSSION THREAD

**Discussion by nick200000** [ywmoyue/biliuwp-lite#758] [feature]
### 问题描述

建议能增加最高音质选项,播放时优先自动选择hires=>无损=>320K=>192K....
现版本,当我在A视频播放时选择无损,再播放其他无无损选项的视频,会跳回132K...
建议能增加选项,优先选择最高音质...谢谢

### 描述解决方案

建议能增加最高音质选项,播放时优先自动选择hires=>无损=>320K=>192K....
现版本,当我在A视频播放时选择无损,再播放其他无无损选项的视频,会跳回132K...
建议能增加选项,优先选择最高音质...谢谢

### 代码贡献

否

**Comment by ywmoyue** [comment]
4.7.3 已提供
